jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -192,7 +192,6 @@ public OutputPathsParameters getOutputPaths() {
   }
 
   @Input
-  @Optional
```

Comment:
I don't know if we have defaults for this parameter, like we do for `useCurrentTimestamp`. Its instantiated with
```	
allowInsecureRegistries = objectFactory.property(Boolean.class);
```